modules/op-enabling-results.adoc

```diff
@@ -3,30 +3,33 @@
 // * cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="installing-results_{context}"]
-= Installing {tekton-results}
+[id="enabling-results_{context}"]
+= Enabling {tekton-results}
 
 [role="_abstract"]
-To install {tekton-results}, you must provide the required resources and then create and apply a `TektonResult` custom resource (CR). The {pipelines-shortname} Operator installs the Results services when you apply the `TektonResult` custom resource.
+The {pipelines-shortname} Operator installs the Results component by default when installint {pipelines-shortname}. To enable {tekton-results}, you must provide the required resources and then make changes in the `TektonConfig` custom resource (CR).
 
 .Prerequisites
 
 * You installed {pipelines-shortname} using the Operator.
 * You prepared a secret with the SSL certificate.
-* You prepared storage for the logging information.
+* You prepared storage or LokiStack forwarding for the logging information.
 * You prepared a secret with the database credentials.
 
 .Procedure
 
-. Create the resource definition file named `result.yaml` based on the following example. You can adjust the settings as necessary.
+. Edit the `TektonResults` CR to make the following changes in the `result` spec.
 +
 [source,yaml]
 ----
-  apiVersion: operator.tekton.dev/v1alpha1
-  kind: TektonResult
-  metadata:
-    name: result
-  spec:
+apiVersion: operator.tekton.dev/v1alpha1
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
```

Comment:
Why did you change this? It should be `TektonResult` and name `result`. Even above we instruct customers to edit `TektonResults` CR